shared/src/iosTest/kotlin/com/twilio/passkeys/TwilioPasskeysTests.kt

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright © 2024 Twilio Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.twilio.passkeys
+
+import com.twilio.passkeys.exception.TwilioException
+import com.twilio.passkeys.mocks.ATTESTATION_OBJECT
+import com.twilio.passkeys.mocks.AUTHENTICATOR_ATTACHMENT
+import com.twilio.passkeys.mocks.AUTHENTICATOR_RESULT_PAYLOAD
+import com.twilio.passkeys.mocks.AuthorizationControllerWrapperMock
+import com.twilio.passkeys.mocks.CLIENT_DATA_JSON_AUTHENTICATE
+import com.twilio.passkeys.mocks.CLIENT_DATA_JSON_CREATE
+import com.twilio.passkeys.mocks.ID
+import com.twilio.passkeys.mocks.RAW_ID
+import com.twilio.passkeys.mocks.SIGNATURE
+import com.twilio.passkeys.mocks.TYPE
+import com.twilio.passkeys.mocks.USER_HANDLE
+import com.twilio.passkeys.mocks.authenticatePayload
+import com.twilio.passkeys.mocks.createPasskeyChallengePayload
+import com.twilio.passkeys.mocks.createResultPayload
+import com.twilio.passkeys.mocks.transports
+import kotlinx.coroutines.test.runTest
+import platform.UIKit.UIWindow
+import kotlin.experimental.ExperimentalNativeApi
+import kotlin.test.Test
+import kotlin.test.assertTrue
+import kotlin.test.fail
+
+@OptIn(ExperimentalNativeApi::class)
+class TwilioPasskeysTests {
+  // region Properties
+  private val passkeyPayloadMapper = PasskeyPayloadMapper
+  private val authControllerWrapper: AuthorizationControllerWrapperMock = AuthorizationControllerWrapperMock()
+  private val twilioPasskey = TwilioPasskeys(passkeyPayloadMapper, authControllerWrapper)
+  // endregion
+
+  // region Tests
+  @Test
+  fun `Test Passkey Creation With A Valid Response Should Succeed`() =
```

Comment:
The test function names should not include `Test` as a prefix since the `@Test` annotation already indicates that it's a test. Instead, we can use a more natural naming style, such as:

`Passkey creation with a valid response should succeed`

Reply:
Fixed, Thank you.